pxr/base/arch/fileSystem.cpp

```diff
@@ -1111,76 +1111,27 @@ int ArchFileAccess(const char* path, int mode)
 
 // https://msdn.microsoft.com/en-us/library/windows/hardware/ff552012.aspx
```

Comment:
Looks like this comment refers to the structure you've deleted below?  If so then the comment should be deleted also.